Google.Api.Gax.Grpc/ClientBuilderBase.cs

```diff
@@ -750,6 +750,22 @@ public async Task<TClient> BuildAsync(IServiceProvider provider, CancellationTok
         protected virtual ChannelBase CreateChannel(string endpoint, ChannelCredentials credentials) =>
             LastCreatedChannel = EffectiveGrpcAdapter.CreateChannel(ServiceMetadata, endpoint, credentials, GetChannelOptions());
 
+        /// <summary>
+        /// Retrives the value of the environment variable with <paramref name="name"/>,
+        /// mapping empty or whitespace-only strings to null.
+        /// </summary>
+        private static string GetNonEmptyOrNullEnvironmentVariable(string name) => GetNonEmptyOrNullValue(name, Environment.GetEnvironmentVariable);
```

Comment:
Maybe GetNonWhiteSpaceOrNull? I'd expected a call to string.IsNullOrEmpty later rather than string.IsNullOrWhiteSpace.

Reply:
Yes, that's better, changing...